src/views/static/community/community-view.js

```diff
@@ -0,0 +1,47 @@
+import { customElement, html, LitElement } from 'lit-element';
```

Comment:
Is this page needed at all? I think it is not being used?

Reply:
I think we want to use these pages in the future, yes. We need to find ways to link the menus to such pages when they are a 'head entry' in the menu. Doesn't work at the moment unfortunately.